forum/users/models.py

```diff
@@ -1,3 +1,59 @@
 from django.db import models
+from django.contrib.auth.models import (
+    AbstractBaseUser,
+    BaseUserManager,
+    PermissionsMixin,
+)
+from phonenumber_field.modelfields import PhoneNumberField
 
-# Create your models here.
+ROLE_CHOICES = [
+    (0, "Unassigned"),
+    (1, "Startup"),
+    (2, "Investor"),
+    (3, "Both")
+]
```

Comment:
Just the recommendation, - replace the tuple with an `Enum` for better readability and maintainability.
```
from enum import IntEnum

class Role(IntEnum):
    UNASSIGNED = 0
    STARTUP = 1
    INVESTOR = 2
    BOTH = 3

ROLE_CHOICES = [(role.value, role.name.capitalize()) for role in Role]
```

This makes it easier to reference roles in code:

```
if user.role == Role.STARTUP:
```